zk/constants.go

```diff
@@ -69,8 +69,9 @@ const (
 )
 
 const (
-	FlagEphemeral = 1
-	FlagSequence  = 2
+	FlagPersistent = 0
```

Comment:
first thanks for replying me. I have been waiting for your reply for long time.  These flags can be used by other applications,such as codis. Let me show some codes in codis, "zkhelper.CreateRecursive(zkConn, dir, "", 0, zkhelper.DefaultDirACLs())"
"0" should be replaced by "FlagPersistent "